example/android/javalib/3-linting/build.mill

```diff
@@ -41,12 +38,12 @@ object app extends AndroidAppModule {
   def androidLintAbortOnError = false
 }
 
-/** See Also: app/AndroidManifest.xml */
+/** See Also: app/src/main/AndroidManifest.xml */
 
 // The `AndroidManifest.xml` file shown above is flawed with hardcoded string linting problem so it can be spoted by running the `androidLintRun` task.
 
-/** See Also: app/resources/values/colors.xml */
-/** See Also: app/resources/values/strings.xml */
+/** See Also: app/src/main/res/values/colors.xml */
```

Comment:
Just to confirm here that android uses `src/main/res/` as the primary resource folder? I thought most Java projects use `resources/`, but maybe Android has different conventions

Reply:
yes this is the default structure

![image](https://github.com/user-attachments/assets/a5e47f1f-4bae-42f7-86d1-348b70a3afa8)